### PR TITLE
Integration Time Adjusted Completeness Merge

### PR DESCRIPTION
**Pull request by deanthedream**
This merge request contains the new Integration Time Adjusted Completeness module and a uniform albedo distribution planet physical model module.

### CODE REVIEW

File: EXOSIMS/Completeness/IntegrationTimeAdjustedCompleteness.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
```
Comment (dsavransky) [inline EXOSIMS/Completeness/IntegrationTimeAdjustedCompleteness.py:1]: global: can we get a unit test for this and subtype completeness?

File: EXOSIMS/PlanetPhysicalModel/uniform_albedo.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+from EXOSIMS.util.vprint import vprint
```
Comment (dsavransky) [inline EXOSIMS/PlanetPhysicalModel/uniform_albedo.py:1]: This class doesn't need to exist.  Let's port the functionality to the prototype's calc_albedo_from_sma method, and just ensure that the prototype's prange is [0.367, 0.367] (or whatever the exact number in the current method is) to protect backwards compatibility.

File: EXOSIMS/Prototypes/PlanetPhysicalModel.py [hunk 3]
```diff
@@ -56,7 +56,7 @@ def __init__(self, cachedir=None, whichPlanetPhaseFunction='lambert', **specs):
         Phis = self.calc_Phi(betas)
         self.betaFunction = PchipInterpolator(-Phis,betas) #the -Phis ensure the function monotonically increases
 
-        return
+        #return
```
Comment (dsavransky) [inline EXOSIMS/Prototypes/PlanetPhysicalModel.py:59]: why not just delete outright?

**Review by dsavransky — CHANGES_REQUESTED** [review]

File: EXOSIMS/Completeness/IntegrationTimeAdjustedCompleteness.py [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+        SubtypeCompleteness.__init__(self, **specs)
+        print('Num Planets AFTER SubtypeComp declaration: ' + str(Nplanets))
+        #Note: This calls target completeness which calculates TL.comp0, a term used for filtering targets based on low completeness values
+        #This executes with 10^8 planets, the default for SubtypeCompleteness. self.Nplanets is updated later here
+        
+        # Number of planets to sample
+        self.Nplanets = int(Nplanets)
+       
+        # get path to completeness interpolant stored in a pickled .comp file
+        self.filename = self.PlanetPopulation.__class__.__name__ + self.PlanetPhysicalModel.__class__.__name__ + self.__class__.__name__ + str(self.Nplanets) + self.PlanetPhysicalModel.whichPlanetPhaseFunction
```
[the remark this answers is not in the document]
Comment (deanthedream) [reply EXOSIMS/Completeness/IntegrationTimeAdjustedCompleteness.py:49]: Lay off this one. There is a longer discussion to be had about caching for this module and what is/what is not needed. Things work as they stand.

**Pull request merged**